### ISSUE TITLE
Client crashes on pressing ESC whilst editing signpost

### ISSUE THREAD

**Issue by TinyDeskEngineer06** [Gollorum/Signpost#89]
In the latest version of this mod for 1.16.5 (as of 2021-12-11) pressing Escape in the signpost UI causes the client to crash to desktop.
Here's the error message:
```
The game crashed whilst keypressed event handler
Error: java.lang.NullPointerException: keyPressed event handler
Exit Code: -1
```

**Comment by Gollorum** [comment]
Hey, sorry for the delay.
In order for me to reproduce this, I'd appreciate if you could give me some more details:
- Does this happen every time or just in some circumstances?
- What other mods are installed?

**Comment by Gollorum** [comment]
Probably fixed with version 2.00.4, plz reopen if it still happens
